FFXIVClientStructs/FFXIV/Client/UI/Misc/RecommendEquipModule.cs

```diff
@@ -9,34 +9,56 @@ namespace FFXIVClientStructs.FFXIV.Client.UI.Misc;
 public unsafe partial struct RecommendEquipModule {
     public static RecommendEquipModule* Instance() => Framework.Instance()->GetUiModule()->GetRecommendEquipModule();
 
-    [FieldOffset(0x00)] public uint Unk00;
-    [FieldOffset(0x04)] public uint SlotCount; // maybe?
-
-    [FieldOffset(0x08)] public InventoryItem* EquippedMainHand;
-    [FieldOffset(0x10)] public InventoryItem* EquippedOffHand;
-    [FieldOffset(0x18)] public InventoryItem* EquippedHead;
-    [FieldOffset(0x20)] public InventoryItem* EquippedBody;
-    [FieldOffset(0x28)] public InventoryItem* EquippedHands;
-    [FieldOffset(0x30)] public InventoryItem* EquippedWaist; // wtf are you still doing here
-    [FieldOffset(0x38)] public InventoryItem* EquippedLegs;
-    [FieldOffset(0x40)] public InventoryItem* EquippedFeet;
-    [FieldOffset(0x48)] public InventoryItem* EquippedEars;
-    [FieldOffset(0x50)] public InventoryItem* EquippedNeck;
-    [FieldOffset(0x58)] public InventoryItem* EquippedWrist;
-    [FieldOffset(0x60)] public InventoryItem* EquippedLeftRing;
-    [FieldOffset(0x68)] public InventoryItem* EquippedRightRing;
-    [FieldOffset(0x70)] public InventoryItem* EquippedSoulCrystal;
-    [FieldOffset(0x78)] public void* Unk00_15;
+    [FieldOffset(0), Obsolete("Use IsUpdating and IsSetupForDifferentClassJob")] public uint Unk0;
```

Comment:
```suggestion
    [FieldOffset(0), Obsolete("Use IsUpdating and IsSetupForDifferentClassJob")] public uint Unk00;
```
Accidental name change